### ISSUE TITLE
[Bug] Discord invite link invalid

### ISSUE THREAD

**Issue by Barmp** [crowbartools/Firebot#2915]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Current Behavior

The Discord link provided in the New Issue menu has expired. 
![image](https://github.com/user-attachments/assets/9ea80744-4d6e-4ac6-b009-52df58b6d234)
![image](https://github.com/user-attachments/assets/381a300d-599c-4398-908c-06d6a39f8eab)

### Expected Behavior

The link to the crowbartools Discord would work.

### Steps To Reproduce

1. Start a New Issue
2. Choose the option to join the Discord server.
3. Be sad, you are not allowed to join. 

### Version

None

### What operating system are you using Firebot on?

Linux Distro

**Comment by Barmp** [comment]
The link in your wiki (https://discord.com/invite/DegW792) works.

**Comment by Barmp** [comment]
Slightly separate issue, the link to your Bluesky profile could get posted alongside the link to Twitter if you're using that now too. 

(https://bsky.app/profile/firebot.app)

**Comment by zunderscore** [comment]
Yeah, this stuff is on our list to update. I was looking at changing the in-app Twitter links to Bluesky earlier tonight, in fact. I just gotta get icons in. Regardless, I might be able to clean this up soon.

**Comment by zunderscore** [comment]
Hey @Barmp, got the Discord link updated in the issue template and we've added Bluesky links everywhere there used to be Twitter links, so that'll go out with the next `master` push. Thanks for letting us know!
